packages/mobile/src/components/core/SectionList.tsx

```diff
@@ -168,9 +163,15 @@ export const SectionList = forwardRef<RNSectionList, SectionListProps>(
       PlayBarChin
     )
 
+    const areSectionsEmpty = sections.every(
```

Comment:
why move into component?

Reply:
to prevent calling twice, since im reffing in two spots, but yeah didn't need to. when i tried to make a const with a single call, i was getting `const areSectionsEmptyyy = areSectionsEmpty(sections)` so to avoid a clever var name, just used it directly, whatever you think tho

Reply:
carry on